quantum/rgb_matrix_animations/rgb_matrix_effects.inc

```diff
@@ -1,4 +1,5 @@
 // Add your new core rgb matrix effect here, order determins enum order, requires "rgb_matrix_animations/ directory
+#include "rgb_matrix_animations/openrgb_direct_anim.h"
```

Comment:
Would prefer that this was at the end of the list, so it doesn't screw up the mode eeprom setting. 

Instead, could set the default mode to the openRGB mode, if the feature is enabled.